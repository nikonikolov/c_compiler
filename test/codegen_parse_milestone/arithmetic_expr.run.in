int zz(int a, int b, int c){
	5*10;
}

