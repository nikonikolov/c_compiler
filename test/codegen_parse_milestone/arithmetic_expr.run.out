FUNCTION : zz
    PARAM : a
    PARAM : b
    PARAM : c
SCOPE
50
